resources/views/welcome.blade.php

```diff
@@ -6,7 +6,8 @@
             <div class="col-md-12">
               <div class="text-center position-relative header-text">
                 <h1 class="display-1 font-weight-bold text-white">Swipe Right</h1>
-                <button class="btn mt-4 text-white rounded-pill px-5 font-weight-bolder" data-toggle="modal" data-target="#registerModal">Créer un compte</button>
+                <button class="btn mt-4 text-white rounded-pill px-5 font-weight-bolder">  <a href="{{ route('register') }}" style="color: white ">Créer un compte</a> </button>
+                {{-- <button class="btn mt-4 text-white rounded-pill px-5 font-weight-bolder" data-toggle="modal" data-target="#registerModal">Créer un compte</button> --}}
```

Comment:
Please delete all the commented code on your files

Reply:
this code comment will have their importance